src/pages/resultsView/coExpression/CoExpressionTab.tsx

```diff
@@ -90,53 +142,112 @@ export default class CoExpressionTab extends React.Component<ICoExpressionTabPro
         );
     }
 
+    @bind
+    private isSelectedGeneticEntityAGeneset(geneticEntity:GeneticEntity) {
```

Comment:
- this is a setter but its named like a getter
- it doesnt use the parameter
- it has the same name as the @computed getter above, really confusing for me

Reply:
I've removed this part, it is not used